homeassistant/components/hardware/models.py

```diff
@@ -17,12 +17,24 @@ class BoardInfo:
     revision: str | None
 
 
+@dataclass
+class USBInfo:
+    """USB info type."""
+
+    vid: str
+    pid: str
+    serial_number: str | None
+    manufacturer: str | None
+    description: str | None
+
+
 @dataclass
 class HardwareInfo:
     """Hardware info type."""
 
     name: str | None
     board: BoardInfo | None
+    dongles: list[USBInfo] | None
```

Comment:
Since it's always either `dongles` or `board`, maybe we should make two subclass? BoardInfo and DongleInfo ?